### PR TITLE
Various changes

### PR DESCRIPTION
**Pull request by sanmadjack**
Another batch of changes and fixes for you to consider

### CODE REVIEW

File: core/polyfills.php [hunk 1]
```diff
@@ -281,6 +281,7 @@ function mb_strtolower($str)
     'mpg' => 'video/mpeg',
     'mpeg' => 'video/mpeg',
     'mov' => 'video/quicktime',
+    'flv' => 'video/x-flv',
```
Comment (shish) [inline core/polyfills.php]: flv is already defined as video/x-flv on line 261

File: ext/approval/info.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * Name: Trash
+ * Author: Matthew Barbour <matthew@darkohlme.net>
+ * Description: Provides "Trash" or "Recycle Bin"-type functionality, storing delete images for later recovery
+ * Documentation:
```
Comment (shish) [inline ext/approval/info.php]: These seem to need updating~ (Do we even need metadata in comments, now that we have metadata in `ExtensionInfo`?)

**Comment by shish** [comment]
Reviewed / tested / accepted most of these; a couple of minor nits and merge conflicts with what's left :3

File: ext/approval/info.php (hunk 2 above)
Comment (sanmadjack) [reply ext/approval/info.php]: I would think not. We can probably just go ahead and delete them all.

File: core/polyfills.php (hunk 1 above)
Comment (sanmadjack) [reply core/polyfills.php]: Weird, it was missing in my copy. Must have been a merge mixup somewhere.

**Comment by sanmadjack** [comment]
I re-based and addressed the noted issues, and tossed on a few more commits

**Pull request merged**